api/Onboard.go

```diff
@@ -0,0 +1,103 @@
+package api
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/operate-first/opfcli/constants"
+	"github.com/operate-first/opfcli/models"
+	"github.com/operate-first/opfcli/utils"
+)
+
+// TODO: use kustomization package so we can unmarshal kustomizations properly, and not just resources or : https://pkg.go.dev/github.com/williepaul/kustomize/api/types
+
+func (api *API) Onboard(path string, projectDisplayName string, DisableLimitrange bool) error {
+	onboardRequest, err := models.OnboardRequestFromYAMLPath(path)
+
+	if err != nil {
+		return err
+	}
+
+	if err := api.CreateGroup(
+		onboardRequest.TeamName,
+		onboardRequest.Users,
+		true,
+	); err != nil {
+		return err
+	}
+
+	if err := api.CreateRoleBinding(onboardRequest.TeamName, "admin"); err != nil {
+		return err
+	}
+
+	var tempQuota string
+	var nsPaths []string
+	emptyCustomQuota := models.CustomResourceQuota{}
+
+	for i := range onboardRequest.Namespaces {
+		if onboardRequest.Namespaces[i].CustomQuota != emptyCustomQuota {
+			tempQuota = ""
+		} else if onboardRequest.Namespaces[i].Quota != "" && onboardRequest.Namespaces[i].Quota != "_No response_" {
+			tempQuota = strings.ToLower(onboardRequest.Namespaces[i].Quota)
+			if err := api.ValidateQuota(tempQuota); err != nil {
+				return err
+			}
+		}
+		if err := api.CreateNamespace(
+			onboardRequest.Namespaces[i].Name,
+			onboardRequest.TeamName,
+			projectDisplayName,
+			tempQuota,
+			DisableLimitrange,
+			true,
+		); err != nil {
+			return err
+		}
+		if onboardRequest.Namespaces[i].CustomQuota != emptyCustomQuota {
+			if err := api.CreateCustomResourceQuota(onboardRequest.Namespaces[i].Name, onboardRequest.Namespaces[i].CustomQuota, true); err != nil {
+				return err
+			}
+		}
+
+		nsPaths = append(nsPaths, fmt.Sprintf("../../../../base/core/namespaces/%s", onboardRequest.Namespaces[i].Name))
+	}
+
+	// core kustomization modifications
```

Comment:
Unclear on "core kustomization modifications". I think you can remove this comment.